packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md

```diff
@@ -0,0 +1,93 @@
+
+What is an Identity Provider? It is a service that creates and maintains identity information and then provides authentication services to your applications. Identity Providers can significantly reduce sign-in and registration friction, which allows your users to access applications with one click and without needing to create new passwords or remember usernames. 
+
+## The big picture
+Okta manages connections to other Identity Providers for your application and sits between your application and the Identity Provider that authenticates your users. 
+
+* When you use Okta as the [user store](https://developer.okta.com/product/user-management/) for your applications, users can sign in with their email and password by default. See our guides for how to sign in users to your [web](/docs/guides/sign-into-web-app/), [mobile](/docs/guides/sign-into-mobile-app/), and [single-page](/docs/guides/sign-into-spa/) apps.
+
+* You can add connections to Identity Providers like Twitter or Facebook. This is called social login or social authentication. It allows your users to sign in to your app using credentials from their existing accounts with those Identity Providers. After users authenticate, you sync their accounts into your Okta Universal Directory while continuing to use that Identity Provider for user authentication. This eliminates the need to store an additional username and password for that user. 
+
+* You can add connections to Identity Providers that you build in-house that support OpenID Connect or SAML protocols, also referred to as inbound federation or inbound SAML. The SAML flow is initiated with the service provider (in this case, Okta) that redirects the user to the Identity Provider for authentication. After authentication, a user is created inside Okta, and the user is redirected back to your application along with an ID token. This allows you to use Okta to proxy between SAML-only Identity Providers and OpenID Connect-only applications that normally are incompatible. 
+
+> Note: Social and OpenID Connect Identity Providers store access tokens that allow subsequent calls to Identity Providers after the user is authorized. For example, the token may contain the permission to add events to a user's Google calendar. After the user is authorized, the token can be used on additional calls to add events to a user's Google calendar on the user's behalf. 
+
+* You can also configure federation between Okta orgs using OpenID Connect or SAML. 
+
+Adding any of these Identity Providers allows users to sign in to your application using their credentials from a specific Identity Provider. 
+
+## Benefits of using Okta to manage Identity Providers
+You could connect your application directly to an Identity Provider (for example, using an SDK to add a button). However, using Okta as the user store for your application and letting Okta manage the Identity Provider connections has some benefits:
```

Comment:
(for example, using an SDK to add a button for **Facebook login**).

?

Reply:
@albertchen-okta  - do we need to specifically call out Facebook here? Does that make it more clear?

Reply:
If we're trying to give an example about  a button called "Sign in with x IDP" then yes I think it makes it more clear, doesn't have to be Facebook, but its makes the point. Otherwise, not sure we need an example here if its self-evident already. Just my 2 cents.